Ascend.java

```diff
@@ -2,6 +2,17 @@
 // and prints them in increasing order.

```

Comment:
OVERALL:
EXCELLENT Submission! great documentation (even if it mainly for yourself)
There are some slight styling mistakes, but overall great! 
KEEP UP THE GREAT WORK